csm/core/services/security.py

```diff
@@ -350,3 +350,37 @@ async def check_certificate_expiry_time_task(self):
                            tzinfo=timezone.utc),
             interval=timedelta(days=1)
         )
+
+    async def _get_name_details(self, name):
+        """
+        Get x509 Name object details (i.e Subject and Issuer)
+        """
+        name_details = {}
+        for k,v in x509.oid._OID_NAMES.items():
+            try:
+                name_details[v] = name.get_attributes_for_oid(k)[0].value
+            except Exception as e:
```

Comment:
`get_attributes_for_oid` returns empty list if nothing matches. Please handle explicit exception like IndexError, AttributError instead of Base exception

Reply:
Done.